.github/workflows/sync-tests.yml

```diff
@@ -1,62 +0,0 @@
-name: sync-tests
-
-on:
-  push:
-    branches:
-      - 'master'
-      - 'develop'
-      - 'regenesis/*'
-  pull_request:
```

Comment:
Perhaps just remove these? We can keep it turned on and use `workflow_dispatch` for it to test manually.

Reply:
^that's a great idea will do